resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -119,12 +119,13 @@ class ClientSuite extends SparkFunSuite with Matchers with BeforeAndAfterAll
       if (LOCAL_SCHEME.equals(uri.getScheme())) {
         cp should contain (uri.getPath())
       } else {
-        cp should not contain (uri.getPath())
+        cp should not contain uri.getPath()
```

Comment:
Could you avoid these noisy changes and also keep the existing style of using parentheses in the code you're actually adding?